package.xml

```diff
@@ -1,24 +1,25 @@
 <?xml version="1.0"?>
 <package format="3">
-    <name>ign_moveit2</name>
-    <version>1.0.0</version>
-    <description>MoveIt2 interface for Ignition Gazebo</description>
-    <maintainer email="orsula.andrej@gmail.com">Andrej Orsula</maintainer>
-    <author>Andrej Orsula</author>
-    <license>BSD</license>
+  <name>ign_moveit2_examples</name>
+  <version>1.0.0</version>
+  <description>C++ and Python examples of using MoveIt 2 for planning motions that are executed inside Ignition Gazebo simulation environment</description>
+  <maintainer email="orsula.andrej@gmail.com">Andrej Orsula</maintainer>
+  <author>Andrej Orsula</author>
+  <license>Apache License 2.0</license>
 
-    <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-    <depend>rclcpp</depend>
-    <depend>moveit_ros_planning_interface</depend>
-    <depend>moveit_msgs</depend>
+  <depend>moveit_msgs</depend>
+  <depend>moveit_ros_planning_interface</depend>
+  <depend>rclcpp</depend>
 
-    <exec_depend>rclpy</exec_depend>
+  <exec_depend>pymoveit2</exec_depend>
+  <exec_depend>rclpy</exec_depend>
```

Comment:
You could add `panda_moveit2_config` as additional exec_depend

Reply:
Oh wait, the package is called `panda_moveit_config` in `panda_ign`

Reply:
Thank you. Yes, I need to add all dependencies into `package.xml` as I have really been neglecting it.

Regarding `panda_moveit2_config`. I have decided to rename the package into `panda_moveit_config` as a part of [panda_ign#7](https://github.com/AndrejOrsula/panda_ign/pull/7) such that it is compatible with current structure in [ros-planning/moveit_resources:ros2](https://github.com/ros-planning/moveit_resources/tree/ros2). https://github.com/AndrejOrsula/panda_moveit2_config is now archived.